focused/security/restBasicAuthCustomStore/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
No license header? Perhaps we want to configure something like mycila license-maven-plugin for consistency?